Tools-Override/tests.targets

```diff
@@ -48,12 +48,10 @@
     <XunitResultsFileName>testResults.xml</XunitResultsFileName>
 
     <XunitOptions Condition="'$(BuildingNETFxVertical)' == 'true'">$(XunitOptions) -noshadow -noappdomain </XunitOptions>
-    <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
+    <XunitOptions Condition="'$(BuildingUAPAOTVertical)' != 'true'">$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
```

Comment:
Writing out testResults.xml require FileStream to be available which is not yet in NetNative's corelib, so disabling for now.